notebook/notebookapp.py

```diff
@@ -548,6 +548,7 @@ def start(self):
     'notebook-dir': 'NotebookApp.notebook_dir',
     'browser': 'NotebookApp.browser',
     'pylab': 'NotebookApp.pylab',
+    'connect-host': 'NotebookApp.connect_host',
```

Comment:
I don't think this is an option that warrants a CLI alias.

Reply:
Removed